internal/pkg/service/common/etcdop/op/atomic.go

```diff
@@ -138,7 +138,7 @@ func (v *AtomicOp) DoWithoutRetry(ctx context.Context, client etcd.KV, opts ...O
 
 	// Create IF part of the transaction
 	var cmps []etcd.Cmp
-	for _, op := range tracker.Operations() {
+	for _, op := range removeOpsOverlaps(tracker.Operations()) {
```

Comment:
Ako som pisal, tracker nam pre iterator reportuje tieto operacie:
```
[some/prefix/ -> ................................................ some/prefix)
                   [some/prefix/foo003 -> ....................... some/prefix)
                                      [some/prefix/foo005 -> .... some/prefix)
```

Pri atomickej operacii na zaklade toho generujeme IF podmienky pre transakciu, ze sa ziadna z tychto range nezmenila.

Maximalny pocet je `128` poloziek, ... tj. ak by sa nacitalo viac `pages`, tak nam to uz nepojde s `AtomicOp`.

Okrem toho je to zbytocne, kedze prva `range` pokryva aj vsetky ostatne.
Preto som tu pridal odstranenie duplikatov, necha sa iba najdlhsia range, a tie co s pokryte sa odstrania.

Vid unit test.